src/lib/djerba/html/genomic_details_template.html

```diff
@@ -69,11 +69,29 @@ <h3 class="header1">Structural Variants and Fusions</h3>
 </tbody>
 </table>
 % endif
-## ends	keep-together
+## ends keep-together
 </div>
 % endif
 <div class="keep-together">
 <div style="height:36px">
+<h3 class="header1">Other Biomarkers</h3>
+</div>
+<div style="height:36px">
+<h3 class="header3">Microsatellite Instability</h3>
+</div>
+<div>
+<table width="100%">
+<tbody>
+<tr>
+<td width="50%"><strong>MSI-H</strong>: this sample likely has genetic or epigenetic alterations resulting in <strong>loss of function of mismatch repair (MMR) genes</strong>, which can lead to a microsatellite instability-high (MSI-H)/mismatch repair deficient (MMR-D) phenotype.</td>
```

Comment:
TMB-H I wasn't going to include because we have TMB in the Genomic Summary part already, and TMB-H will show up in Clinically Actionable. As for MSI-H, you are right that there should be some kind of change for MSI-H vs MSS. Thoughts and recommendations on the best way to do that? An if statement directly in the mako or should I write two different texts elsewhere that get pasted into the template depending on specifics of case?

Reply:
If at all possible this should be handled further upstream -- can we have the MSI-H vs MSS text in the JSON? Then the template will simply write the appropriate text in a table.